src/tabbar.h

```diff
@@ -50,6 +50,7 @@ class TabBar : public QTabBar
     void openFindInPageBar();
     void closeTabsByZimId(const QString &id);
     QStringList getTabUrls() const;
+    QStringList getTabZimIds() const;
```

Comment:
This function should be introduced in the next commit.

Reply:
Done